Index: xterm/xterm.man
===================================================================
--- xterm.orig/xterm.man
+++ xterm/xterm.man
@@ -1514,7 +1514,7 @@
 .B "allowWindowOps (\fPclass\fB AllowWindowOps)"
 Specifies whether extended window control sequences (as used in dtterm)
 should be allowed.
-The default is ``true.''
+The default is ``false.''
 .TP 8
 .B "altIsNotMeta (\fPclass\fB AltIsNotMeta\fP)"
 If ``true'', treat the Alt-key as if it were the Meta-key.
Index: xterm/charproc.c
===================================================================
--- xterm.orig/charproc.c
+++ xterm/charproc.c
@@ -392,7 +392,7 @@
     Bres(XtNallowFontOps, XtCAllowFontOps, screen.allowFontOp0, True),
     Bres(XtNallowTcapOps, XtCAllowTcapOps, screen.allowTcapOp0, False),
     Bres(XtNallowTitleOps, XtCAllowTitleOps, screen.allowTitleOp0, True),
-    Bres(XtNallowWindowOps, XtCAllowWindowOps, screen.allowWindowOp0, True),
+    Bres(XtNallowWindowOps, XtCAllowWindowOps, screen.allowWindowOp0, False),
     Bres(XtNaltIsNotMeta, XtCAltIsNotMeta, screen.alt_is_not_meta, False),
     Bres(XtNaltSendsEscape, XtCAltSendsEscape, screen.alt_sends_esc, False),
     Bres(XtNalwaysBoldMode, XtCAlwaysBoldMode, screen.always_bold_mode, False),
