Index: xterm-242/charproc.c
===================================================================
--- xterm-242.orig/charproc.c
+++ xterm-242/charproc.c
@@ -395,7 +395,7 @@
 static XtResource resources[] =
 {
     Bres(XtNallowSendEvents, XtCAllowSendEvents, screen.allowSendEvent0, False),
-    Bres(XtNallowFontOps, XtCAllowFontOps, screen.allowFontOp0, True),
+    Bres(XtNallowFontOps, XtCAllowFontOps, screen.allowFontOp0, False),
     Bres(XtNallowTcapOps, XtCAllowTcapOps, screen.allowTcapOp0, False),
     Bres(XtNallowTitleOps, XtCAllowTitleOps, screen.allowTitleOp0, True),
     Bres(XtNallowWindowOps, XtCAllowWindowOps, screen.allowWindowOp0, False),
Index: xterm-242/xterm.man
===================================================================
--- xterm-242.orig/xterm.man
+++ xterm-242/xterm.man
@@ -1474,7 +1474,7 @@
 .TP
 .B "allowFontOps (\fPclass\fB AllowFontOps)"
 Specifies whether control sequences that set/query the font should be allowed.
-The default is ``true.''
+The default is ``false.''
 .TP 8
 .B "allowSendEvents (\fPclass\fB AllowSendEvents)"
 Specifies whether or not synthetic key and button events (generated using
