---
 xterm.man |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/xterm.man
+++ b/xterm.man
@@ -1899,7 +1899,7 @@ The default is \*(``true\*(''.
 .TP
 .B "allowFontOps (\fPclass\fB AllowFontOps)"
 Specifies whether control sequences that set/query the font should be allowed.
-The default is \*(``true\*(''.
+The default is \*(``false\*(''.
 .TP 8
 .B "allowPasteControls (\fPclass\fB AllowPasteControls)"
 If true, allow control characters such as BEL and CAN to be pasted.
