---
 xterm.man |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/xterm.man
+++ b/xterm.man
@@ -1997,7 +1997,7 @@ The default is \*(``true\*(''.
 .TP
 .B "allowFontOps\fP (class\fB AllowFontOps\fP)"
 Specifies whether control sequences that set/query the font should be allowed.
-The default is \*(``true\*(''.
+The default is \*(``false\*(''.
 .TP 8
 .B "allowPasteControls\fP (class\fB AllowPasteControls\fP)"
 If true, allow control characters such as BEL and CAN to be pasted.
